lib/chat_api/slack.ex

```diff
@@ -0,0 +1,44 @@
+defmodule ChatApi.Slack do
+  @moduledoc """
+  A module to handle sending Slack notifications.
+  """
+
+  use Tesla
+
+  plug Tesla.Middleware.BaseUrl, "https://hooks.slack.com"
+  plug Tesla.Middleware.Headers, [{"content-type", "application/json"}]
+  plug Tesla.Middleware.JSON
+
+  def send(message) do
+    webhook_url = System.get_env("PAPERCUPS_SLACK_WEBHOOK_URL")
+
+    if is_nil(webhook_url) do
+      raise "The SLACK_WEBHOOK_URL environment variable is required to send Slack messages"
+    end
+
+    post(webhook_url, message)
+  end
+
+  def send_conversation_message_alert(conversation_id, text) do
+    # TODO: handle this in user settings, for now just toggling on/off with env
+    slack_enabled = System.get_env("PAPERCUPS_SLACK_ENABLED")
+    url = "https://www.papercups.io/conversations/" <> conversation_id
+    description = "conversation " <> conversation_id
+    # Oof interpolation is a little awkward, but this creates a formatted link
+    # e.g. `<papercups.io/conversations/xxxxx|New message in conversation xxxxx>`
+    link = "<" <> url <> "|" <> description <> ">"
```

Comment:
"<#{url}|#{description}>" is probably cleaner

Reply:
oh nice didn't realize we could do that lol